<!DOCTYPE html>
<html lang="en"><head>
<script src="index_files/libs/clipboard/clipboard.min.js"></script>
<script src="index_files/libs/quarto-html/tabby.min.js"></script>
<script src="index_files/libs/quarto-html/popper.min.js"></script>
<script src="index_files/libs/quarto-html/tippy.umd.min.js"></script>
<link href="index_files/libs/quarto-html/tippy.css" rel="stylesheet">
<link href="index_files/libs/quarto-html/light-border.css" rel="stylesheet">
<link href="index_files/libs/quarto-html/quarto-html.min.css" rel="stylesheet" data-mode="light">
<link href="index_files/libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles"><meta charset="utf-8">
  <meta name="generator" content="quarto-1.3.450">

  <meta name="author" content="John Vivian, Ph.D">
  <meta name="dcterms.date" content="2024-03-14">
  <title>Single-cell sequencing of the blood T cell repertoire</title>
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent">
  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no, minimal-ui">
  <link rel="stylesheet" href="index_files/libs/revealjs/dist/reset.css">
  <link rel="stylesheet" href="index_files/libs/revealjs/dist/reveal.css">
  <style>
    code{white-space: pre-wrap;}
    span.smallcaps{font-variant: small-caps;}
    div.columns{display: flex; gap: min(4vw, 1.5em);}
    div.column{flex: auto; overflow-x: auto;}
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
    ul.task-list{list-style: none;}
    ul.task-list li input[type="checkbox"] {
      width: 0.8em;
      margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
      vertical-align: middle;
    }
  </style>
  <link rel="stylesheet" href="index_files/libs/revealjs/dist/theme/quarto.css">
  <link rel="stylesheet" href="styles.css">
  <link href="index_files/libs/revealjs/plugin/quarto-line-highlight/line-highlight.css" rel="stylesheet">
  <link href="index_files/libs/revealjs/plugin/reveal-menu/menu.css" rel="stylesheet">
  <link href="index_files/libs/revealjs/plugin/reveal-menu/quarto-menu.css" rel="stylesheet">
  <link href="index_files/libs/revealjs/plugin/quarto-support/footer.css" rel="stylesheet">
  <style type="text/css">

  .callout {
    margin-top: 1em;
    margin-bottom: 1em;  
    border-radius: .25rem;
  }

  .callout.callout-style-simple { 
    padding: 0em 0.5em;
    border-left: solid #acacac .3rem;
    border-right: solid 1px silver;
    border-top: solid 1px silver;
    border-bottom: solid 1px silver;
    display: flex;
  }

  .callout.callout-style-default {
    border-left: solid #acacac .3rem;
    border-right: solid 1px silver;
    border-top: solid 1px silver;
    border-bottom: solid 1px silver;
  }

  .callout .callout-body-container {
    flex-grow: 1;
  }

  .callout.callout-style-simple .callout-body {
    font-size: 1rem;
    font-weight: 400;
  }

  .callout.callout-style-default .callout-body {
    font-size: 0.9rem;
    font-weight: 400;
  }

  .callout.callout-titled.callout-style-simple .callout-body {
    margin-top: 0.2em;
  }

  .callout:not(.callout-titled) .callout-body {
      display: flex;
  }

  .callout:not(.no-icon).callout-titled.callout-style-simple .callout-content {
    padding-left: 1.6em;
  }

  .callout.callout-titled .callout-header {
    padding-top: 0.2em;
    margin-bottom: -0.2em;
  }

  .callout.callout-titled .callout-title  p {
    margin-top: 0.5em;
    margin-bottom: 0.5em;
  }
    
  .callout.callout-titled.callout-style-simple .callout-content  p {
    margin-top: 0;
  }

  .callout.callout-titled.callout-style-default .callout-content  p {
    margin-top: 0.7em;
  }

  .callout.callout-style-simple div.callout-title {
    border-bottom: none;
    font-size: .9rem;
    font-weight: 600;
    opacity: 75%;
  }

  .callout.callout-style-default  div.callout-title {
    border-bottom: none;
    font-weight: 600;
    opacity: 85%;
    font-size: 0.9rem;
    padding-left: 0.5em;
    padding-right: 0.5em;
  }

  .callout.callout-style-default div.callout-content {
    padding-left: 0.5em;
    padding-right: 0.5em;
  }

  .callout.callout-style-simple .callout-icon::before {
    height: 1rem;
    width: 1rem;
    display: inline-block;
    content: "";
    background-repeat: no-repeat;
    background-size: 1rem 1rem;
  }

  .callout.callout-style-default .callout-icon::before {
    height: 0.9rem;
    width: 0.9rem;
    display: inline-block;
    content: "";
    background-repeat: no-repeat;
    background-size: 0.9rem 0.9rem;
  }

  .callout-title {
    display: flex
  }
    
  .callout-icon::before {
    margin-top: 1rem;
    padding-right: .5rem;
  }

  .callout.no-icon::before {
    display: none !important;
  }

  .callout.callout-titled .callout-body > .callout-content > :last-child {
    margin-bottom: 0.5rem;
  }

  .callout.callout-titled .callout-icon::before {
    margin-top: .5rem;
    padding-right: .5rem;
  }

  .callout:not(.callout-titled) .callout-icon::before {
    margin-top: 1rem;
    padding-right: .5rem;
  }

  /* Callout Types */

  div.callout-note {
    border-left-color: #4582ec !important;
  }

  div.callout-note .callout-icon::before {
    background-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAYAAABzenr0AAAAAXNSR0IArs4c6QAAAERlWElmTU0AKgAAAAgAAYdpAAQAAAABAAAAGgAAAAAAA6ABAAMAAAABAAEAAKACAAQAAAABAAAAIKADAAQAAAABAAAAIAAAAACshmLzAAAEU0lEQVRYCcVXTWhcVRQ+586kSUMMxkyaElstCto2SIhitS5Ek8xUKV2poatCcVHtUlFQk8mbaaziwpWgglJwVaquitBOfhQXFlqlzSJpFSpIYyXNjBNiTCck7x2/8/LeNDOZxDuEkgOXe++553zfefee+/OYLOXFk3+1LLrRdiO81yNqZ6K9cG0P3MeFaMIQjXssE8Z1JzLO9ls20MBZX7oG8w9GxB0goaPrW5aNMp1yOZIa7Wv6o2ykpLtmAPs/vrG14Z+6d4jpbSKuhdcSyq9wGMPXjonwmESXrriLzFGOdDBLB8Y6MNYBu0dRokSygMA/mrun8MGFN3behm6VVAwg4WR3i6FvYK1T7MHo9BK7ydH+1uurECoouk5MPRyVSBrBHMYwVobG2aOXM07sWrn5qgB60rc6mcwIDJtQrnrEr44kmy+UO9r0u9O5/YbkS9juQckLed3DyW2XV/qWBBB3ptvI8EUY3I9p/67OW+g967TNr3Sotn3IuVlfMLVnsBwH4fsnebJvyGm5GeIUA3jljERmrv49SizPYuq+z7c2H/jlGC+Ghhupn/hcapqmcudB9jwJ/3jvnvu6vu5lVzF1fXyZuZZ7U8nRmVzytvT+H3kilYvH09mLWrQdwFSsFEsxFVs5fK7A0g8gMZjbif4ACpKbjv7gNGaD8bUrlk8x+KRflttr22JEMRUbTUwwDQScyzPgedQHZT0xnx7ujw2jfVfExwYHwOsDTjLdJ2ebmeQIlJ7neo41s/DrsL3kl+W2lWvAga0tR3zueGr6GL78M3ifH0rGXrBC2aAR8uYcIA5gwV8zIE8onoh8u0Fca/ciF7j1uOzEnqcIm59sEXoGc0+z6+H45V1CvAvHcD7THztu669cnp+L0okAeIc6zjbM/24LgGM1gZk7jnRu1aQWoU9sfUOuhrmtaPIO3YY1KLLWZaEO5TKUbMY5zx8W9UJ6elpLwKXbsaZ4EFl7B4bMtDv0iRipKoDQT2sNQI9b1utXFdYisi+wzZ/ri/1m7QfDgEuvgUUEIJPq3DhX/5DWNqIXDOweC2wvIR90Oq3lDpdMIgD2r0dXvGdsEW5H6x6HLRJYU7C69VefO1x8Gde1ZFSJLfWS1jbCnhtOPxmpfv2LXOA2Xk2tvnwKKPFuZ/oRmwBwqRQDcKNeVQkYcOjtWVBuM/JuYw5b6isojIkYxyYAFn5K7ZBF10fea52y8QltAg6jnMqNHFBmGkQ1j+U43HMi2xMar1Nv0zGsf1s8nUsmUtPOOrbFIR8bHFDMB5zL13Gmr/kGlCkUzedTzzmzsaJXhYawnA3UmARpiYj5ooJZiUoxFRtK3X6pgNPv+IZVPcnwbOl6f+aBaO1CNvPW9n9LmCp01nuSaTRF2YxHqZ8DYQT6WsXT+RD6eUztwYLZ8rM+rcPxamv1VQzFUkzFXvkiVrySGQgJNvXHJAxiU3/NwiC03rSf05VBaPtu/Z7/B8Yn/w7eguloAAAAAElFTkSuQmCC');
  }

  div.callout-note.callout-style-default .callout-title {
    background-color: #dae6fb
  }

  div.callout-important {
    border-left-color: #d9534f !important;
  }

  div.callout-important .callout-icon::before {
    background-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAYAAABzenr0AAAAAXNSR0IArs4c6QAAAERlWElmTU0AKgAAAAgAAYdpAAQAAAABAAAAGgAAAAAAA6ABAAMAAAABAAEAAKACAAQAAAABAAAAIKADAAQAAAABAAAAIAAAAACshmLzAAAEKklEQVRYCcVXTWhcVRS+575MJym48A+hSRFr00ySRQhURRfd2HYjk2SSTokuBCkU2o0LoSKKraKIBTcuFCoidGFD08nkBzdREbpQ1EDNIv8qSGMFUboImMSZd4/f9zJv8ibJMC8xJQfO3HPPPef7zrvvvnvviIkpC9nsw0UttFunbUhpFzFtarSd6WJkStVMw5xyVqYTvkwfzuf/5FgtkVoB0729j1rjXwThS7Vio+Mo6DNnvLfahoZ+i/o32lULuJ3NNiz7q6+pyAUkJaFF6JwaM2lUJlV0MlnQn5aTRbEu0SEqHUa0A4AdiGuB1kFXRfVyg5d87+Dg4DL6m2TLAub60ilj7A1Ec4odSAc8X95sHh7+ZRPCFo6Fnp7HfU/fBng/hi10CjCnWnJjsxvDNxWw0NfV6Rv5GgP3I3jGWXumdTD/3cbEOP2ZbOZp69yniG3FQ9z1jD7bnBu9Fc2tKGC2q+uAJOQHBDRiZX1x36o7fWBs7J9ownbtO+n0/qWkvW7UPIfc37WgT6ZGR++EOJyeQDSb9UB+DZ1G6DdLDzyS+b/kBCYGsYgJbSQHuThGKRcw5xdeQf8YdNHsc6ePXrlSYMBuSIAFTGAtQo+VuALo4BX83N190NWZWbynBjhOHsmNfFWLeL6v+ynsA58zDvvAC8j5PkbOcXCMg2PZFk3q8MjI7WAG/Dp9AwP7jdGBOOQkAvlFUB+irtm16I1Zw9YBcpGTGXYmk3kQIC/Cds55l+iMI3jqhjAuaoe+am2Jw5GT3Nbz3CkE12NavmzN5+erJW7046n/CH1RO/RVa8lBLozXk9uqykkGAyRXLWlLv5jyp4RFsG5vGVzpDLnIjTWgnRy2Rr+tDKvRc7Y8AyZq10jj8DqXdnIRNtFZb+t/ZRtXcDiVnzpqx8mPcDWxgARUqx0W1QB9MeUZiNrV4qP+Ehc+BpNgATsTX8ozYKL2NtFYAHc84fG7ndxUPr+AR/iQSns7uSUufAymwDOb2+NjK27lEFocm/EE2WpyIy/Hi66MWuMKJn8RvxIcj87IM5Vh9663ziW36kR0HNenXuxmfaD8JC7tfKbrhFr7LiZCrMjrzTeGx+PmkosrkNzW94ObzwocJ7A1HokLolY+AvkTiD/q1H0cN48c5EL8Crkttsa/AXQVDmutfyku0E7jShx49XqV3MFK8IryDhYVbj7Sj2P2eBxwcXoe8T8idsKKPRcnZw1b+slFTubwUwhktrfnAt7J++jwQtLZcm3sr9LQrjRzz6cfMv9aLvgmnAGvpoaGLxM4mAEaLV7iAzQ3oU0IvD5x9ix3yF2RAAuYAOO2f7PEFWCXZ4C9Pb2UsgDeVnFSpbFK7/IWu7TPTvBqzbGdCHOJQSxiEjt6IyZmxQyEJHv6xyQsYk//moVFsN2zP6fRImjfq7/n/wFDguUQFNEwugAAAABJRU5ErkJggg==');
  }

  div.callout-important.callout-style-default .callout-title {
    background-color: #f7dddc
  }

  div.callout-warning {
    border-left-color: #f0ad4e !important;
  }

  div.callout-warning .callout-icon::before {
    background-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAYAAABzenr0AAAAAXNSR0IArs4c6QAAAERlWElmTU0AKgAAAAgAAYdpAAQAAAABAAAAGgAAAAAAA6ABAAMAAAABAAEAAKACAAQAAAABAAAAIKADAAQAAAABAAAAIAAAAACshmLzAAAETklEQVRYCeVWW2gcVRg+58yaTUnizqbipZeX4uWhBEniBaoUX1Ioze52t7sRq6APio9V9MEaoWlVsFasRq0gltaAPuxms8lu0gcviE/FFOstVbSIxgcv6SU7EZqmdc7v9+9mJtNks51NTUH84ed889/PP+cmxP+d5FIbMJmNbpREu4WUkiTtCicKny0l1pIKmBzovF2S+hIJHX8iEu3hZJ5lNZGqyRrGSIQpq15AzF28jgpeY6yk6GVdrfFqdrD6Iw+QlB8g0YS2g7dyQmXM/IDhBhT0UCiRf59lfqmmDvzRt6kByV/m4JjtzuaujMUM2c5Z2d6JdKrRb3K2q6mA+oYVz8JnDdKPmmNthzkAk/lN63sYPgevrguc72aZX/L9C6x09GYyxBgCX4NlvyGUHOKELlm5rXeR1kchuChJt4SSwyddZRXgvwMGvYo4QSlk3/zkHD8UHxwVJA6zjZZqP8v8kK8OWLnIZtLyCAJagYC4rTGW/9Pqj92N/c+LUaAj27movwbi19tk/whRCIE7Q9vyI6yvRpftAKVTdUjOW40X3h5OXsKCdmFcx0xlLJoSuQngnrJe7Kcjm4OMq9FlC7CMmScQANuNvjfP3PjGXDBaUQmbp296S5L4DrpbrHN1T87ZVEZVCzg1FF0Ft+dKrlLukI+/c9ENo+TvlTDbYFvuKPtQ9+l052rXrgKoWkDAFnvh0wTOmYn8R5f4k/jN/fZiCM1tQx9jQQ4ANhqG4hiL0qIFTGViG9DKB7GYzgubnpofgYRwO+DFjh0Zin2m4b/97EDkXkc+f6xYAPX0KK2I/7fUQuwzuwo/L3AkcjugPNixC8cHf0FyPjWlItmLxWw4Ou9YsQCr5fijMGoD/zpdRy95HRysyXA74MWOnscpO4j2y3HAVisw85hX5+AFBRSHt4ShfLFkIMXTqyKFc46xdzQM6XbAi702a7sy04J0+feReMFKp5q9esYLCqAZYw/k14E/xcLLsFElaornTuJB0svMuJINy8xkIYuL+xPAlWRceH6+HX7THJ0djLUom46zREu7tTkxwmf/FdOZ/sh6Q8qvEAiHpm4PJ4a/doJe0gH1t+aHRgCzOvBvJedEK5OFE5jpm4AGP2a8Dxe3gGJ/pAutug9Gp6he92CsSsWBaEcxGx0FHytmIpuqGkOpldqNYQK8cSoXvd+xLxXADw0kf6UkJNFtdo5MOgaLjiQOQHcn+A6h5NuL2s0qsC2LOM75PcF3yr5STuBSAcGG+meA14K/CI21HcS4LBT6tv0QAh8Dr5l93AhZzG5ZJ4VxAqdZUEl9z7WJ4aN+svMvwHHL21UKTd1mqvChH7/Za5xzXBBKrUcB0TQ+Ulgkfbi/H/YT5EptrGzsEK7tR1B7ln9BBwckYfMiuSqklSznIuoIIOM42MQO+QnduCoFCI0bpkzjCjddHPN/F+2Yu+sd9bKNpVwHhbS3LluK/0zgfwD0xYI5dXuzlQAAAABJRU5ErkJggg==');
  }

  div.callout-warning.callout-style-default .callout-title {
    background-color: #fcefdc
  }

  div.callout-tip {
    border-left-color: #02b875 !important;
  }

  div.callout-tip .callout-icon::before {
    background-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAYAAABzenr0AAAAAXNSR0IArs4c6QAAAERlWElmTU0AKgAAAAgAAYdpAAQAAAABAAAAGgAAAAAAA6ABAAMAAAABAAEAAKACAAQAAAABAAAAIKADAAQAAAABAAAAIAAAAACshmLzAAADr0lEQVRYCe1XTWgTQRj9ZjZV8a9SPIkKgj8I1bMHsUWrqYLVg4Ue6v9BwZOxSYsIerFao7UiUryIqJcqgtpimhbBXoSCVxUFe9CTiogUrUp2Pt+3aUI2u5vdNh4dmMzOzHvvezuz8xNFM0mjnbXaNu1MvFWRXkXEyE6aYOYJpdW4IXuA4r0fo8qqSMDBU0v1HJUgVieAXxzCsdE/YJTdFcVIZQNMyhruOMJKXYFoLfIfIvVIMWdsrd+Rpd86ZmyzzjJmLStqRn0v8lzkb4rVIXvnpScOJuAn2ACC65FkPzEdEy4TPWRLJ2h7z4cArXzzaOdKlbOvKKX25Wl00jSnrwVxAg3o4dRxhO13RBSdNvH0xSARv3adTXbBdTf64IWO2vH0LT+cv4GR1DJt+DUItaQogeBX/chhbTBxEiZ6gftlDNXTrvT7co4ub5A6gp9HIcHvzTa46OS5fBeP87Qm0fQkr4FsYgVQ7Qg+ZayaDg9jhg1GkWj8RG6lkeSacrrHgDaxdoBiZPg+NXV/KifMuB6//JmYH4CntVEHy/keA6x4h4CU5oFy8GzrBS18cLJMXcljAKB6INjWsRcuZBWVaS3GDrqB7rdapVIeA+isQ57Eev9eCqzqOa81CY05VLd6SamW2wA2H3SiTbnbSxmzfp7WtKZkqy4mdyAlGx7ennghYf8voqp9cLSgKdqNfa6RdRsAAkPwRuJZNbpByn+RrJi1RXTwdi8RQF6ymDwGMAtZ6TVE+4uoKh+MYkcLsT0Hk8eAienbiGdjJHZTpmNjlbFJNKDVAp2fJlYju6IreQxQ08UJDNYdoLSl6AadO+fFuCQqVMB1NJwPm69T04Wv5WhfcWyfXQB+wXRs1pt+nCknRa0LVzSA/2B+a9+zQJadb7IyyV24YAxKp2Jqs3emZTuNnKxsah+uabKbMk7CbTgJx/zIgQYErIeTKRQ9yD9wxVof5YolPHqaWo7TD6tJlh7jQnK5z2n3+fGdggIOx2kaa2YI9QWarc5Ce1ipNWMKeSG4DysFF52KBmTNMmn5HqCFkwy34rDg05gDwgH3bBi+sgFhN/e8QvRn8kbamCOhgrZ9GJhFDgfcMHzFb6BAtjKpFhzTjwv1KCVuxHvCbsSiEz4CANnj84cwHdFXAbAOJ4LTSAawGWFn5tDhLMYz6nWeU2wJfIhmIJBefcd/A5FWQWGgrWzyORZ3Q6HuV+Jf0Bj+BTX69fm1zWgK7By1YTXchFDORywnfQ7GpzOo6S+qECrsx2ifVQAAAABJRU5ErkJggg==');
  }

  div.callout-tip.callout-style-default .callout-title {
    background-color: #ccf1e3
  }

  div.callout-caution {
    border-left-color: #fd7e14 !important;
  }

  div.callout-caution .callout-icon::before {
    background-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAYAAABzenr0AAAAAXNSR0IArs4c6QAAAERlWElmTU0AKgAAAAgAAYdpAAQAAAABAAAAGgAAAAAAA6ABAAMAAAABAAEAAKACAAQAAAABAAAAIKADAAQAAAABAAAAIAAAAACshmLzAAACV0lEQVRYCdVWzWoUQRCuqp2ICBLJXgITZL1EfQDBW/bkzUMUD7klD+ATSHBEfAIfQO+iXsWDxJsHL96EHAwhgzlkg8nBg25XWb0zIb0zs9muYYWkoKeru+vn664fBqElyZNuyh167NXJ8Ut8McjbmEraKHkd7uAnAFku+VWdb3reSmRV8PKSLfZ0Gjn3a6Xlcq9YGb6tADjn+lUfTXtVmaZ1KwBIvFI11rRXlWlatwIAAv2asaa9mlB9wwygiDX26qaw1yYPzFXg2N1GgG0FMF8Oj+VIx7E/03lHx8UhvYyNZLN7BwSPgekXXLribw7w5/c8EF+DBK5idvDVYtEEwMeYefjjLAdEyQ3M9nfOkgnPTEkYU+sxMq0BxNR6jExrAI31H1rzvLEfRIdgcv1XEdj6QTQAS2wtstEALLG1yEZ3QhH6oDX7ExBSFEkFINXH98NTrme5IOaaA7kIfiu2L8A3qhH9zRbukdCqdsA98TdElyeMe5BI8Rs2xHRIsoTSSVFfCFCWGPn9XHb4cdobRIWABNf0add9jakDjQJpJ1bTXOJXnnRXHRf+dNL1ZV1MBRCXhMbaHqGI1JkKIL7+i8uffuP6wVQAzO7+qVEbF6NbS0LJureYcWXUUhH66nLR5rYmva+2tjRFtojkM2aD76HEGAD3tPtKM309FJg5j/K682ywcWJ3PASCcycH/22u+Bh7Aa0ehM2Fu4z0SAE81HF9RkB21c5bEn4Dzw+/qNOyXr3DCTQDMBOdhi4nAgiFDGCinIa2owCEChUwD8qzd03PG+qdW/4fDzjUMcE1ZpIAAAAASUVORK5CYII=');
  }

  div.callout-caution.callout-style-default .callout-title {
    background-color: #ffe5d0
  }

  </style>
  <style type="text/css">
    .reveal div.sourceCode {
      margin: 0;
      overflow: auto;
    }
    .reveal div.hanging-indent {
      margin-left: 1em;
      text-indent: -1em;
    }
    .reveal .slide:not(.center) {
      height: 100%;
    }
    .reveal .slide.scrollable {
      overflow-y: auto;
    }
    .reveal .footnotes {
      height: 100%;
      overflow-y: auto;
    }
    .reveal .slide .absolute {
      position: absolute;
      display: block;
    }
    .reveal .footnotes ol {
      counter-reset: ol;
      list-style-type: none; 
      margin-left: 0;
    }
    .reveal .footnotes ol li:before {
      counter-increment: ol;
      content: counter(ol) ". "; 
    }
    .reveal .footnotes ol li > p:first-child {
      display: inline-block;
    }
    .reveal .slide ul,
    .reveal .slide ol {
      margin-bottom: 0.5em;
    }
    .reveal .slide ul li,
    .reveal .slide ol li {
      margin-top: 0.4em;
      margin-bottom: 0.2em;
    }
    .reveal .slide ul[role="tablist"] li {
      margin-bottom: 0;
    }
    .reveal .slide ul li > *:first-child,
    .reveal .slide ol li > *:first-child {
      margin-block-start: 0;
    }
    .reveal .slide ul li > *:last-child,
    .reveal .slide ol li > *:last-child {
      margin-block-end: 0;
    }
    .reveal .slide .columns:nth-child(3) {
      margin-block-start: 0.8em;
    }
    .reveal blockquote {
      box-shadow: none;
    }
    .reveal .tippy-content>* {
      margin-top: 0.2em;
      margin-bottom: 0.7em;
    }
    .reveal .tippy-content>*:last-child {
      margin-bottom: 0.2em;
    }
    .reveal .slide > img.stretch.quarto-figure-center,
    .reveal .slide > img.r-stretch.quarto-figure-center {
      display: block;
      margin-left: auto;
      margin-right: auto; 
    }
    .reveal .slide > img.stretch.quarto-figure-left,
    .reveal .slide > img.r-stretch.quarto-figure-left  {
      display: block;
      margin-left: 0;
      margin-right: auto; 
    }
    .reveal .slide > img.stretch.quarto-figure-right,
    .reveal .slide > img.r-stretch.quarto-figure-right  {
      display: block;
      margin-left: auto;
      margin-right: 0; 
    }
  </style>
</head>
<body class="quarto-light">
  <div class="reveal">
    <div class="slides">

<section id="title-slide" data-background-color="black" data-background-image="imgs/cover.png" data-background-opacity="0.2" data-background-size="contain" class="quarto-title-block center">
  <h1 class="title">Single-cell sequencing of the blood T cell repertoire</h1>
  <p class="subtitle">before and after trastuzumab treatment in early-stage HER2+ breast cancer</p>

<div class="quarto-title-authors">
<div class="quarto-title-author">
<div class="quarto-title-author-name">
John Vivian, Ph.D 
</div>
</div>
</div>

  <p class="date">March 14, 2024</p>
</section>
<section>
<section id="introduction" class="title-slide slide level1 center" data-background-color="#f0f4f7">
<h1>Introduction</h1>

</section>
<section id="section" class="slide level2 center" data-background-iframe="https://jvivian.github.io/">
<h2></h2>
</section></section>
<section>
<section id="career-timeline" class="title-slide slide level1 center" data-background-color="#f0f4f7" data-auto-animate="true">
<h1 data-id="quarto-animate-title">Career Timeline</h1>
<div class="columns">
<div class="column" style="width:20%;">

</div><div class="column" style="width:60%;">
<p><img data-src="imgs/timeline.png"></p>
</div>
</div>
</section>
<section id="nanopore-research" class="slide level2 center" data-background-color="#f0f4f7" data-auto-animate="true">
<h2 data-id="quarto-animate-title">Nanopore Research</h2>
<div class="columns">
<div class="column" style="width:15%;">

</div><div class="column" style="width:85%;">
<p><img data-src="imgs/timeline_nanopore.png"></p>
</div>
</div>
</section>
<section id="nanopore-reread-project" class="slide level2 center" data-transition="zoom">
<h2>Nanopore reread project</h2>

<img data-src="imgs/nanopore_figure.png" style="width:100.0%" class="r-stretch quarto-figure-center"><p class="caption"><strong>A</strong>: Substrate bound to the lipid bilayer via a cholesterol tag with a helicase (Hel308) bound and inactive at the other end. <strong>B</strong>: Electric potential causes the DNA duplex to unwind, leaving behind the cholesterol tether bound to the complement strand. <strong>C</strong>: Electric potential and the steady-state nature of the G-Quadruplex (GQ) causes it to unfold allowing the helicase to become enzymatically active in the 3’ -&gt; 5’ direction. <strong>D</strong>: As Hel308 translocates the DNA back up through the pore, the GQ will refold allowing an additional enzyme to become bound. <strong>E</strong>: Continued translocation. <strong>F</strong>: More than 3-4 abasic residues causes Hel308 to dissociate from the substrate (9 are used in the substrate). The DNA strand is then pulled back down and the cycle is able to repeat itself once more.</p></section>
<section id="computational-genomics-lab" class="slide level2 center" data-background-color="#f0f4f7" data-auto-animate="true">
<h2 data-id="quarto-animate-title">Computational Genomics Lab</h2>
<div class="columns">
<div class="column" style="width:20%;">

</div><div class="column" style="width:80%;">
<p><img data-src="imgs/timeline_grad.png"></p>
</div>
</div>
<!-- ## Computational Genomics Lab {background-color=#f0f4f7 auto-animate="true"}
:::{.columns}
::::{.column width="50%"}
![](imgs/timeline_grad.png)
::::
::::{.column width="50%"}
![](imgs/papers.png)
::::
::: -->
<!-- ## {background-image=imgs/toil_splash.png} -->
</section>
<section id="section-1" class="slide level2 center" data-background-iframe="https://toil.ucsc-cgl.org/">
<h2></h2>
</section>
<section id="section-2" class="slide level2 center" data-background-image="imgs/toil-splash-page.png" data-background-opacity="0.25" data-transition="zoom">
<h2></h2>
<div class="columns">
<div class="column" style="width:5%;">

</div><div class="column" style="width:50%;">
<p><img data-src="imgs/toil.png"></p>
</div>
</div>
</section>
<section id="data-access-gtex-and-ucsc-genome-browser" class="slide level2 center" data-background-image="imgs/toil-splash-page.png" data-background-opacity="0.1" data-transition="convex" data-auto-animate="true">
<h2 data-id="quarto-animate-title">Data Access: GTEx and UCSC Genome Browser</h2>
<div class="columns">
<div class="column" style="width:55%;">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="imgs/gtex.png"></p>
<figcaption><strong>UCSC Genome Browser</strong>: A detailed 6700 bp exon-focused snapshot of a 43 Kbp segment on human chromosome 17 from the UCSC Genome Browser and highlights the tissue-specific expression of two genes as identified by GTEx RNA-seq. Tracks show tissue-specific expression of two genes from the GTEx dataset. 33 samples and 5 tissues displayed out of 7,304 samples and 53 tissues. The tracks for hg19 were produced using the UCSC liftOver tool, which converts coordinates from hg38 bedGraph files created by STAR2 within the Toil pipeline.</figcaption>
</figure>
</div>
</div><div class="column" style="width:45%;">
<p><img data-src="imgs/UCSC-Genome-browser-paper.png"><br></p>
</div>
</div>
</section>
<section id="data-access-gtex-and-ucsc-genome-browser-1" class="slide level2 center" data-background-image="imgs/toil-splash-page.png" data-background-opacity="0.1" data-transition="convex" data-auto-animate="true">
<h2 data-id="quarto-animate-title">Data Access: GTEx and UCSC Genome Browser</h2>
<div class="columns">
<div class="column" style="width:50%;">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="imgs/gtex.png"></p>
<figcaption><strong>UCSC Genome Browser</strong>: A detailed 6700 bp exon-focused snapshot of a 43 Kbp segment on human chromosome 17 from the UCSC Genome Browser and highlights the tissue-specific expression of two genes as identified by GTEx RNA-seq. Tracks show tissue-specific expression of two genes from the GTEx dataset. 33 samples and 5 tissues displayed out of 7,304 samples and 53 tissues. The tracks for hg19 were produced using the UCSC liftOver tool, which converts coordinates from hg38 bedGraph files created by STAR2 within the Toil pipeline.</figcaption>
</figure>
</div>
</div><div class="column" style="width:50%;">
<p><img data-src="imgs/sc_gtex_paper.png"><br></p>
</div>
</div>
</section>
<section id="section-3" class="slide level2 center" data-background-iframe="https://xena.ucsc.edu">
<h2></h2>
</section>
<section id="section-4" class="slide level2 center" data-background-image="imgs/ucsc-xena-splash.png" data-background-opacity="0.2" data-transition="zoom">
<h2></h2>
<div class="columns">
<div class="column" style="width:10%;">

</div><div class="column" style="width:50%;">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="imgs/xena.png"></p>
<figcaption><strong>UCSC Xena Browser</strong>: FOXM1 and its isoforms across 19,948 <strong>TCGA, TARGET, GTEx</strong> samples, noting upregulation of FOXM1B and C in tumors versus GTEx normals using UCSC Xena Browser</figcaption>
</figure>
</div>
</div>
</div>
<aside class="notes">
<ul>
<li>Collaborated with researchers in the ucsc genome browser team to create RNA-seq expression browser tracks</li>
<li>Developed the toil-RNAseq pipeline</li>
<li>TCAP, known as the titin cap protein, is predominantly expressed in muscle tissue. On the other hand, PP1R1B, which is a potential therapeutic target for neurological disorders, is expressed in the brain’s basal ganglia but not in muscle or brain cortex</li>
<li>Collaborated with the UCSC Xena team to provide an open-source data exploration platform of all ~20,000 samples</li>
</ul>
<style type="text/css">
        span.MJX_Assistive_MathML {
          position:absolute!important;
          clip: rect(1px, 1px, 1px, 1px);
          padding: 1px 0 0 0!important;
          border: 0!important;
          height: 1px!important;
          width: 1px!important;
          overflow: hidden!important;
          display:block!important;
      }</style></aside>
</section>
<section id="research-data-collaborations" class="slide level2 center" data-transition="concave">
<h2>Research Data Collaborations</h2>

<!-- ## {transition="zoom" background-image="imgs/UCSC-treehouse-splash.png" background-opacity=0.3 auto-animate="true"}
:::{.columns}
::::{.column width="20%"}
::::
::::{.column width="80%"}
![](imgs/jco_paper_title.png)
::::
::: -->
<img data-src="imgs/UCSC-treehouse-splash.png" class="r-stretch"></section>
<section id="section-5" class="slide level2 center" data-transition="zoom" data-background-image="imgs/UCSC-treehouse-splash.png" data-background-opacity="0.3">
<h2></h2>
<div class="columns">
<div class="column" style="width:51%;">
<p><img data-src="imgs/jco_paper_title.png"></p>
</div><div class="column" style="width:35%;">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="imgs/jco_plate_notation.jpeg"></p>
<figcaption>The model represented in Bayesian plate notation. <strong>G</strong> and <strong>D</strong> stand for Gene and Dataset respectively. <span class="math inline">\(x\)</span> denotes the gene expression from a background dataset, forming a convex combination <span class="math inline">\(\hat{y}\)</span> when multiplied by the dot product of <span class="math inline">\(\beta^T\)</span>. The model incorporates a ‘jointly Dirichlet’ distribution for <span class="math inline">\(\beta\)</span>, as each dataset contributes to the overall distribution. We add a Laplacian error term <span class="math inline">\(\epsilon\)</span> to the expected expression <span class="math inline">\(\hat{y}\)</span> when modeling the observed expression of a new sample <span class="math inline">\(y\)</span>.</figcaption>
</figure>
</div>
</div>
</div>
</section>
<section id="atreca-inc." class="slide level2 center" data-background-color="#f0f4f7" data-auto-animate="true">
<h2 data-id="quarto-animate-title">Atreca Inc.&nbsp;</h2>
<div class="columns">
<div class="column" style="width:25%;">

</div><div class="column" style="width:75%;">
<p><img data-src="imgs/timeline_atreca.png"></p>
</div>
</div>
<aside class="notes">
<ul>
<li>Mention collaborating closely with protein engineering to design, maintain, improve tools they used every day</li>
<li>Ab-CRUNCH
<ul>
<li>Annotated reports of antibody sequences, metadata, chain sequences, multiple alignments, and more</li>
<li>200 antibodies took 45 - 60 minutes to process, after I rewrote it from scratch I reduced the codebase by 66% and 200 antibodies could be processed and returned in about a minute.</li>
</ul></li>
</ul>
<style type="text/css">
        span.MJX_Assistive_MathML {
          position:absolute!important;
          clip: rect(1px, 1px, 1px, 1px);
          padding: 1px 0 0 0!important;
          border: 0!important;
          height: 1px!important;
          width: 1px!important;
          overflow: hidden!important;
          display:block!important;
      }</style></aside>
</section>
<section id="novel-antibody-target-discovery-platform" class="slide level2 center" data-auto-animate="true" data-background-color="#f0f4f7">
<h2 data-id="quarto-animate-title">Novel Antibody Target Discovery Platform</h2>
<div class="columns">
<div class="column" style="width:60%;">
<p><img data-src="imgs/atreca_paper.png"></p>
</div><div class="column" style="width:40%;">
<p><img data-src="imgs/timeline_atreca.png"></p>
</div>
</div>
</section>
<section id="teiko-bio" class="slide level2 center" data-background-color="#f0f4f7" data-auto-animate="true">
<h2 data-id="quarto-animate-title">Teiko Bio</h2>
<div class="columns">
<div class="column" style="width:15%;">

</div><div class="column" style="width:85%;">
<p><img data-src="imgs/timeline_teiko.png"></p>
</div>
</div>
</section>
<section id="next-steps" class="slide level2 center" data-background-color="#f0f4f7" data-auto-animate="true">
<h2 data-id="quarto-animate-title">Next Steps?</h2>
<div class="columns">
<div class="column" style="width:5%;">

</div><div class="column" style="width:75%;">
<p><img data-src="imgs/timeline_ideaya.png"></p>
</div>
</div>
</section></section>
<section>
<section id="analysis-of-t-cell-sub-populations-in-her2-breast-cancer" class="title-slide slide level1 center" data-background-color="#f0f4f7">
<h1>Analysis of T-cell sub-populations in HER2+ breast cancer</h1>

</section>
<section id="study-background" class="slide level2 center" data-background-color="#f0f4f7">
<h2>Study Background</h2>
<div class="columns">
<div class="column" style="width:50%;">
<div class="center">
<p><strong>HER2+ Breast Cancer</strong></p>
</div>
<ul>
<li>HER2-positive breast cancer accounts for approximately 20% of breast cancer cases and is characterized by the overexpression of the human epidermal growth factor receptor 2 (HER2) gene <sup>1</sup></li>
</ul>
<div>
<ul>
<li class="fragment">Anti-HER2 treatments, such as trastuzumab, have drastically the management of HER2-positive breast cancer, turning a once difficult-to-treat cancer into one with much more favorable outcomes <sup>2</sup></li>
</ul>
</div>
</div><div class="column" style="width:50%;">
<div>
<div class="center">
<p><strong>Tumor-Infiltrating Lymphocytes</strong></p>
</div>
<ul>
<li class="fragment">Anti-tumor immunity plays a role in the efficacy of anti-HER2 treatment, as demonstrated by the association of TILs with prognosis and increased efficacy from treatment with trastuzumab and chemotherapy in HER2+ breast cancer <sup>3</sup></li>
<li class="fragment">Immunohistochemistry and RNA sequencing of breast tumor tissues has suggested that trastuzumab may recruit and activate anti-tumor T cells <sup>4</sup></li>
</ul>
</div>
</div>
</div>
<aside><ol class="aside-footnotes"><li id="fn1"><p>Slamon, D. J., et al.&nbsp;(2001). “Use of chemotherapy plus a monoclonal antibody against HER2 for metastatic breast cancer that overexpresses HER2.” New England Journal of Medicine, 344(11), 783-792.</p></li><li id="fn2"><p>Piccart-Gebhart, M. J., et al.&nbsp;(2005). “Trastuzumab after adjuvant chemotherapy in HER2-positive breast cancer.” New England Journal of Medicine, 353(16), 1659-1672.</p></li><li id="fn3"><p>Loi, S., et al.&nbsp;(2014). “Tumor infiltrating lymphocytes are prognostic in triple negative breast cancer and predictive for trastuzumab benefit in early breast cancer.” Ann Oncol, 25(8), 1544–1550.</p></li><li id="fn4"><p>Hendricks, WPD., et al.&nbsp;(2019). “PD-1-Associated Gene Expression Signature of Neoadjuvant Trastuzumab-Treated Tumors Correlates with Patient Survival in HER2-Positive Breast Cancer.” Cancers, 11(10), 156.</p></li></ol></aside></section>
<section id="study-design" class="slide level2 center" data-auto-animate="true">
<h2 data-id="quarto-animate-title">Study Design</h2>
<div class="columns">
<div class="column" style="width:15%;">

</div><div class="column" style="width:85%;">
<p><img data-src="imgs/sc_fig1.png"></p>
</div>
</div>
</section>
<section id="study-design-1" class="slide level2 center" data-auto-animate="true">
<h2 data-id="quarto-animate-title">Study Design</h2>
<div class="columns">
<div class="column" style="width:35%;">
<div class="center">
<p><strong>Patient Cohort and Treatment Regimen</strong></p>
</div>
<ul>
<li><strong>Study Group</strong>: Analysis involved paired pre-treatment and on-treatment PBMC samples from five patients with HER2+ breast cancer, staged IIA–IV (IIA [n=2], IIIC [n=2], IV [n=1])</li>
<li><strong>Treatment Protocol</strong>: Patients received preoperative therapy comprising docetaxel, carboplatin, trastuzumab, and pertuzumab</li>
</ul>
</div><div class="column" style="width:65%;">
<p><img data-src="imgs/sc_fig1.png"></p>
</div>
</div>
</section>
<section id="study-design-2" class="slide level2 center" data-auto-animate="true">
<h2 data-id="quarto-animate-title">Study Design</h2>
<div class="columns">
<div class="column" style="width:40%;">
<div class="center">
<p><strong>Sampling and Analytical Methods</strong></p>
</div>
<ul>
<li><strong>Sample Collection</strong>: Blood samples were collected at the start of Cycle 1, Day 1 (C1D1) for pre-treatment baseline and on Day 1 of Cycle 3, 4, or 5 (C3, C4, C5) to monitor treatment response</li>
<li><strong>Cellular Profiling</strong>: Peripheral CD3+ T cells from these samples were subjected to detailed profiling, utilizing 10x Genomics VDJ alongside gene expression single-cell sequencing to elucidate the immune landscape and gene expression alterations induced by the treatment</li>
</ul>
</div><div class="column" style="width:60%;">
<p><img data-src="imgs/sc_fig1.png"></p>
</div>
</div>
</section>
<section id="vdj-pipeline" class="slide level2 center" data-auto-animate="true">
<h2 data-id="quarto-animate-title">VDJ Pipeline</h2>

<img data-src="imgs/sc_vdj.png" class="r-stretch quarto-figure-center"><p class="caption">The cellranger vdj pipeline can be used to analyze sequencing data produced from Chromium Next GEM Single Cell 5’ V(D)J libraries. It takes FASTQ files for V(D)J libraries and performs sequence assembly and paired clonotype calling. The pipeline uses the Chromium Cell Barcodes (also called 10x Barcodes) and UMIs to assemble V(D)J transcripts per cell.</p></section>
<section id="cite-seq" class="slide level2 center" data-auto-animate="true" data-transition="convex">
<h2 data-id="quarto-animate-title">Cite-Seq</h2>
<div class="columns">
<div class="column" style="width:45%;">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="imgs/sc_vdj.png"></p>
<figcaption>The cellranger vdj pipeline can be used to analyze sequencing data produced from Chromium Next GEM Single Cell 5’ V(D)J libraries. It takes FASTQ files for V(D)J libraries and performs sequence assembly and paired clonotype calling. The pipeline uses the Chromium Cell Barcodes (also called 10x Barcodes) and UMIs to assemble V(D)J transcripts per cell.</figcaption>
</figure>
</div>
</div><div class="column" style="width:55%;">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="imgs/cite-seq.png"></p>
<figcaption>CITE-seq uses DNA-barcoded antibodies to convert detection of proteins into a quantitative, sequenceable readout. Antibody-bound oligos act as synthetic transcripts that are captured during most large-scale oligodT-based scRNA-seq library preparation protocols (e.g.&nbsp;10x Genomics, Drop-seq, ddSeq).</figcaption>
</figure>
</div>
</div>
</div>
</section>
<section id="single-cell-data" class="slide level2 center">
<h2>Single-Cell Data</h2>
<div class="columns">
<div class="column" style="width:50%;">
<ul>
<li><strong>Cellular Data Depth</strong>: High-quality data with over 6,500 cells profiled per sample</li>
</ul>
<div>
<ul>
<li class="fragment"><strong>Gene Expression</strong>: Approximately 41 million reads per sample, ensuring robust gene expression profiling. TCR Analysis: Achieved ~100,000 Unique Molecular Identifiers (UMIs) per sample for precise T-cell receptor (TCR) sequencing</li>
<li class="fragment"><strong>CITE-Seq</strong>: CITE-seq marker panel was included and used in conjunction with gene expression QC to filter the dataset and improve downstream analysis</li>
</ul>
</div>
</div><div class="column" style="width:50%;">
<p><img data-src="imgs/sc_cell_counts.png"></p>
</div>
</div>
</section>
<section id="computational-methods" class="slide level2 center">
<h2>Computational Methods</h2>
<div class="columns">
<div class="column" style="width:50%;">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="imgs/anndata_schema.svg"></p>
<figcaption><code>AnnData</code> schema for storing annotated data. The core data structure used by <code>scanpy</code> which was used for this analysis.</figcaption>
</figure>
</div>
</div><div class="column" style="width:50%;">
<p><br><br></p>
<div class="center">
<p><strong>Data Processing</strong></p>
</div>
<ul>
<li><strong>Filter Cells</strong> - Remove cells with high mitochondrial content and too few genes detected</li>
<li><strong>Filter Genes</strong> - Remove genes below minimum cell and variance threshold</li>
<li><strong>Normalization</strong> - Regress out effects by total count and scale each gene to unit variance</li>
</ul>
</div>
</div>
</section>
<section id="batch-correction-and-dimensionality-reduction" class="slide level2 center">
<h2>Batch Correction and Dimensionality Reduction</h2>

<img data-src="imgs/Harmony.jpeg" class="r-stretch quarto-figure-center"><p class="caption"><code>Harmony</code> operates in embedding space and produces a corrected <code>PCA</code> embedding which is used downstream to calculate the neighborhood graph which in turn is used by <code>UMAP</code> for further dimnsionality reduction.</p></section>
<section id="clustering-and-annotation" class="slide level2 center">
<h2>Clustering and Annotation</h2>
<div class="columns">
<div class="column" style="width:40%;">
<p><br><br></p>
<div class="center">
<p><strong>CD8+ and CD4+ T-cell subsets</strong></p>
</div>
<ul>
<li>Eleven T cell subsets were characterized based on transcriptional profiles from the peripheral blood of donors</li>
<li>Undefined (grey) represents a collection of cells that did not confidently correspond to signatures of collected T cell subsets</li>
</ul>
</div><div class="column" style="width:60%;">
<p><img data-src="imgs/sc_umap.png"></p>
</div>
</div>
<!-- ## Clonal expansions in patients with pathological complete response -->
</section>
<section id="patient-level-analysis" class="slide level2 center">
<h2>Patient-level Analysis</h2>

<img data-src="imgs/sc_umap_by_patient.png" class="r-stretch quarto-figure-center"><p class="caption">Large T cell clonal expansions were observed in pre- and on-treatment peripheral blood samples from two patients who exhibited a pathological complete response. A high abundance of activated CD8+ TEM/EM cells was seen in patients 1 and 5 (red arrows). The other three patients had partial clinical responses with residual disease at surgery. Patient 4 had an increase in activated CD4+ Tregs on-treatment (red circle)</p></section>
<section id="variation-in-t-cell-populations-across-treatment-stages" class="slide level2 center" data-auto-animate="true">
<h2 data-id="quarto-animate-title">Variation in T-Cell Populations Across Treatment Stages</h2>
<div class="columns">
<div class="column" style="width:35%;">
<p><br><br></p>
<ul>
<li>Clonal expansions were observed in activated CD8+ TEM/EM cells in patients 1 and 5</li>
</ul>
<div>
<ul>
<li class="fragment">Clonal expansion of activated CD4+ TEM/EM cells was observed in patient 5</li>
<li class="fragment">No significant expansions were observed in the other subsets</li>
</ul>
</div>
</div><div class="column" style="width:65%;">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="imgs/sc_tcell_subsets.png"></p>
<figcaption>All five samples had similar pre- and on-treatment subset distributions and clonal expansion profiles</figcaption>
</figure>
</div>
</div>
</div>
</section>
<section id="variation-in-t-cell-populations-across-treatment-stages-1" class="slide level2 center" data-auto-animate="true">
<h2 data-id="quarto-animate-title">Variation in T-Cell Populations Across Treatment Stages</h2>

<img data-src="imgs/sc_population_over_time.png" class="r-stretch quarto-figure-center"><p class="caption">A possible trend toward on-treatment increase in activated CD4+ Tregs was observed. Activated CD4+ Treg subset sizes are small and therefore estimation of abundance has high error bars.</p></section>
<section id="summary" class="slide level2 center" data-background-color="#f0f4f7">
<h2>Summary</h2>
<ul>
<li>Single-cell mRNA sequencing enables high-resolution characterization of immune cell subsets and represents a promising approach to assess the immune response to anti-HER2 treatment and other cancer therapeutics</li>
</ul>
<div>
<ul>
<li class="fragment">In this study, single-cell mRNA sequencing of peripheral CD3+ T-cells identified clonal expansions in activated CD8+ and CD4+ T cells in HER2+ breast cancer patients who had a pathological complete response with preoperative docetaxel, carboplatin, trastuzumab, and pertuzumab treatment</li>
<li class="fragment">These data are consistent with the model that T-cells play a key role in anti-HER2-mediated tumor control and warrant further investigation in a larger sample population</li>
</ul>
</div>
<div class="fragment">
<ul>
<li>Due to the limited sample size (<span class="math inline">\(n=5\)</span>), it was challenging to establish statistical correlations between complete pathological responses and the relative proportions of activated CD4/CD8 T-cell populations. Bayesian power analysis can be used to estimate the number of samples we <em>would</em> need to reliably quantify a given effect size.</li>
</ul>
</div>
</section></section>
<section>
<section id="open-for-questions" class="title-slide slide level1 center" data-background-color="#f0f4f7">
<h1>Open for Questions</h1>
<p>Thank you for your time and consideration</p>
</section>
<section id="addendum-bayesian-power-analysis" class="slide level2 smaller center">
<h2>Addendum: Bayesian Power Analysis</h2>
<div class="columns">
<div class="column" style="width:50%;">
<p><br></p>
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="imgs/power_analysis.svg"></p>
<figcaption>This heatmap shows Bayesian power analysis outcomes, where the x-axis corresponds to <strong>sample size</strong> and the y-axis to <strong>effect size</strong>. Each cell shows the <strong>power</strong> (<span class="math inline">\(p \in [0, 1]\)</span>), which corresponds to the proportion of simulations detecting a significant difference. Color intensity denotes power levels, with darker colors signifying higher statistical power. Power analysis can help discern study feasibility as well as provide a sense of what effect size is detectable given sample size limitations.</figcaption>
</figure>
</div>
</div><div class="column" style="width:40%;">
<div class="center">
<p><strong>General Steps</strong></p>
</div>
<ul>
<li>Define your prior beliefs about the parameters of interest</li>
<li>Specify the data generating process model. This model should represent the belief of how data arises within an experimental context</li>
<li>Determine the decision rule defining “significant” or “practically relevant.” A Bayesian approach is to specify a region of practical equivalence (ROPE) around a null value where effects are considered too small to be of interest</li>
<li>Simulate data of varying sample sizes at differing effect sizes</li>
<li>Run inference (e.g.&nbsp;MCMC) on each simulated dataset to estimate the posterior distributions</li>
<li>Calculate power by determining how often the decision rule leads to a conclusion of practical significance by calculating the proportion of simulations where the posterior distribution of the effect size meets your criteria for significance (e.g., falls outside the ROPE).</li>
</ul>
</div>
</div>
<div class="footer footer-default">

</div>
</section></section>

    </div>
  </div>

  <script>window.backupDefine = window.define; window.define = undefined;</script>
  <script src="index_files/libs/revealjs/dist/reveal.js"></script>
  <!-- reveal.js plugins -->
  <script src="index_files/libs/revealjs/plugin/quarto-line-highlight/line-highlight.js"></script>
  <script src="index_files/libs/revealjs/plugin/pdf-export/pdfexport.js"></script>
  <script src="index_files/libs/revealjs/plugin/reveal-menu/menu.js"></script>
  <script src="index_files/libs/revealjs/plugin/reveal-menu/quarto-menu.js"></script>
  <script src="index_files/libs/revealjs/plugin/quarto-support/support.js"></script>
  

  <script src="index_files/libs/revealjs/plugin/notes/notes.js"></script>
  <script src="index_files/libs/revealjs/plugin/search/search.js"></script>
  <script src="index_files/libs/revealjs/plugin/zoom/zoom.js"></script>
  <script src="index_files/libs/revealjs/plugin/math/math.js"></script>
  <script>window.define = window.backupDefine; window.backupDefine = undefined;</script>

  <script>

      // Full list of configuration options available at:
      // https://revealjs.com/config/
      Reveal.initialize({
'controlsAuto': false,
'previewLinksAuto': false,
'smaller': false,
'pdfSeparateFragments': false,
'autoAnimateEasing': "ease",
'autoAnimateDuration': 1,
'autoAnimateUnmatched': true,
'menu': {"side":"left","useTextContentForMissingTitles":true,"markers":false,"loadIcons":false,"custom":[{"title":"Tools","icon":"<i class=\"fas fa-gear\"></i>","content":"<ul class=\"slide-menu-items\">\n<li class=\"slide-tool-item active\" data-item=\"0\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.fullscreen(event)\"><kbd>f</kbd> Fullscreen</a></li>\n<li class=\"slide-tool-item\" data-item=\"1\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.speakerMode(event)\"><kbd>s</kbd> Speaker View</a></li>\n<li class=\"slide-tool-item\" data-item=\"2\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.overview(event)\"><kbd>o</kbd> Slide Overview</a></li>\n<li class=\"slide-tool-item\" data-item=\"3\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.togglePdfExport(event)\"><kbd>e</kbd> PDF Export Mode</a></li>\n<li class=\"slide-tool-item\" data-item=\"4\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.keyboardHelp(event)\"><kbd>?</kbd> Keyboard Help</a></li>\n</ul>"}],"openButton":true},
'smaller': false,
 
        // Display controls in the bottom right corner
        controls: true,

        // Help the user learn the controls by providing hints, for example by
        // bouncing the down arrow when they first encounter a vertical slide
        controlsTutorial: false,

        // Determines where controls appear, "edges" or "bottom-right"
        controlsLayout: 'edges',

        // Visibility rule for backwards navigation arrows; "faded", "hidden"
        // or "visible"
        controlsBackArrows: 'faded',

        // Display a presentation progress bar
        progress: true,

        // Display the page number of the current slide
        slideNumber: false,

        // 'all', 'print', or 'speaker'
        showSlideNumber: 'all',

        // Add the current slide number to the URL hash so that reloading the
        // page/copying the URL will return you to the same slide
        hash: true,

        // Start with 1 for the hash rather than 0
        hashOneBasedIndex: false,

        // Flags if we should monitor the hash and change slides accordingly
        respondToHashChanges: true,

        // Push each slide change to the browser history
        history: true,

        // Enable keyboard shortcuts for navigation
        keyboard: true,

        // Enable the slide overview mode
        overview: true,

        // Disables the default reveal.js slide layout (scaling and centering)
        // so that you can use custom CSS layout
        disableLayout: false,

        // Vertical centering of slides
        center: true,

        // Enables touch navigation on devices with touch input
        touch: true,

        // Loop the presentation
        loop: false,

        // Change the presentation direction to be RTL
        rtl: false,

        // see https://revealjs.com/vertical-slides/#navigation-mode
        navigationMode: 'linear',

        // Randomizes the order of slides each time the presentation loads
        shuffle: false,

        // Turns fragments on and off globally
        fragments: true,

        // Flags whether to include the current fragment in the URL,
        // so that reloading brings you to the same fragment position
        fragmentInURL: false,

        // Flags if the presentation is running in an embedded mode,
        // i.e. contained within a limited portion of the screen
        embedded: false,

        // Flags if we should show a help overlay when the questionmark
        // key is pressed
        help: true,

        // Flags if it should be possible to pause the presentation (blackout)
        pause: true,

        // Flags if speaker notes should be visible to all viewers
        showNotes: false,

        // Global override for autoplaying embedded media (null/true/false)
        autoPlayMedia: null,

        // Global override for preloading lazy-loaded iframes (null/true/false)
        preloadIframes: null,

        // Number of milliseconds between automatically proceeding to the
        // next slide, disabled when set to 0, this value can be overwritten
        // by using a data-autoslide attribute on your slides
        autoSlide: 0,

        // Stop auto-sliding after user input
        autoSlideStoppable: true,

        // Use this method for navigation when auto-sliding
        autoSlideMethod: null,

        // Specify the average time in seconds that you think you will spend
        // presenting each slide. This is used to show a pacing timer in the
        // speaker view
        defaultTiming: null,

        // Enable slide navigation via mouse wheel
        mouseWheel: false,

        // The display mode that will be used to show slides
        display: 'block',

        // Hide cursor if inactive
        hideInactiveCursor: true,

        // Time before the cursor is hidden (in ms)
        hideCursorTime: 5000,

        // Opens links in an iframe preview overlay
        previewLinks: false,

        // Transition style (none/fade/slide/convex/concave/zoom)
        transition: 'slide',

        // Transition speed (default/fast/slow)
        transitionSpeed: 'default',

        // Transition style for full page slide backgrounds
        // (none/fade/slide/convex/concave/zoom)
        backgroundTransition: 'none',

        // Number of slides away from the current that are visible
        viewDistance: 3,

        // Number of slides away from the current that are visible on mobile
        // devices. It is advisable to set this to a lower number than
        // viewDistance in order to save resources.
        mobileViewDistance: 2,

        // The "normal" size of the presentation, aspect ratio will be preserved
        // when the presentation is scaled to fit different resolutions. Can be
        // specified using percentage units.
        width: 1728,

        height: 1117,

        // Factor of the display size that should remain empty around the content
        margin: 0.1,

        math: {
          mathjax: 'https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.0/MathJax.js',
          config: 'TeX-AMS_HTML-full',
          tex2jax: {
            inlineMath: [['\\(','\\)']],
            displayMath: [['\\[','\\]']],
            balanceBraces: true,
            processEscapes: false,
            processRefs: true,
            processEnvironments: true,
            preview: 'TeX',
            skipTags: ['script','noscript','style','textarea','pre','code'],
            ignoreClass: 'tex2jax_ignore',
            processClass: 'tex2jax_process'
          },
        },

        // reveal.js plugins
        plugins: [QuartoLineHighlight, PdfExport, RevealMenu, QuartoSupport,

          RevealMath,
          RevealNotes,
          RevealSearch,
          RevealZoom
        ]
      });
    </script>
    <script id="quarto-html-after-body" type="application/javascript">
    window.document.addEventListener("DOMContentLoaded", function (event) {
      const toggleBodyColorMode = (bsSheetEl) => {
        const mode = bsSheetEl.getAttribute("data-mode");
        const bodyEl = window.document.querySelector("body");
        if (mode === "dark") {
          bodyEl.classList.add("quarto-dark");
          bodyEl.classList.remove("quarto-light");
        } else {
          bodyEl.classList.add("quarto-light");
          bodyEl.classList.remove("quarto-dark");
        }
      }
      const toggleBodyColorPrimary = () => {
        const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
        if (bsSheetEl) {
          toggleBodyColorMode(bsSheetEl);
        }
      }
      toggleBodyColorPrimary();  
      const tabsets =  window.document.querySelectorAll(".panel-tabset-tabby")
      tabsets.forEach(function(tabset) {
        const tabby = new Tabby('#' + tabset.id);
      });
      const isCodeAnnotation = (el) => {
        for (const clz of el.classList) {
          if (clz.startsWith('code-annotation-')) {                     
            return true;
          }
        }
        return false;
      }
      const clipboard = new window.ClipboardJS('.code-copy-button', {
        text: function(trigger) {
          const codeEl = trigger.previousElementSibling.cloneNode(true);
          for (const childEl of codeEl.children) {
            if (isCodeAnnotation(childEl)) {
              childEl.remove();
            }
          }
          return codeEl.innerText;
        }
      });
      clipboard.on('success', function(e) {
        // button target
        const button = e.trigger;
        // don't keep focus
        button.blur();
        // flash "checked"
        button.classList.add('code-copy-button-checked');
        var currentTitle = button.getAttribute("title");
        button.setAttribute("title", "Copied!");
        let tooltip;
        if (window.bootstrap) {
          button.setAttribute("data-bs-toggle", "tooltip");
          button.setAttribute("data-bs-placement", "left");
          button.setAttribute("data-bs-title", "Copied!");
          tooltip = new bootstrap.Tooltip(button, 
            { trigger: "manual", 
              customClass: "code-copy-button-tooltip",
              offset: [0, -8]});
          tooltip.show();    
        }
        setTimeout(function() {
          if (tooltip) {
            tooltip.hide();
            button.removeAttribute("data-bs-title");
            button.removeAttribute("data-bs-toggle");
            button.removeAttribute("data-bs-placement");
          }
          button.setAttribute("title", currentTitle);
          button.classList.remove('code-copy-button-checked');
        }, 1000);
        // clear code selection
        e.clearSelection();
      });
      function tippyHover(el, contentFn) {
        const config = {
          allowHTML: true,
          content: contentFn,
          maxWidth: 500,
          delay: 100,
          arrow: false,
          appendTo: function(el) {
              return el.closest('section.slide') || el.parentElement;
          },
          interactive: true,
          interactiveBorder: 10,
          theme: 'light-border',
          placement: 'bottom-start'
        };
          config['offset'] = [0,0];
          config['maxWidth'] = 700;
        window.tippy(el, config); 
      }
      const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
      for (var i=0; i<noterefs.length; i++) {
        const ref = noterefs[i];
        tippyHover(ref, function() {
          // use id or data attribute instead here
          let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
          try { href = new URL(href).hash; } catch {}
          const id = href.replace(/^#\/?/, "");
          const note = window.document.getElementById(id);
          return note.innerHTML;
        });
      }
      const findCites = (el) => {
        const parentEl = el.parentElement;
        if (parentEl) {
          const cites = parentEl.dataset.cites;
          if (cites) {
            return {
              el,
              cites: cites.split(' ')
            };
          } else {
            return findCites(el.parentElement)
          }
        } else {
          return undefined;
        }
      };
      var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
      for (var i=0; i<bibliorefs.length; i++) {
        const ref = bibliorefs[i];
        const citeInfo = findCites(ref);
        if (citeInfo) {
          tippyHover(citeInfo.el, function() {
            var popup = window.document.createElement('div');
            citeInfo.cites.forEach(function(cite) {
              var citeDiv = window.document.createElement('div');
              citeDiv.classList.add('hanging-indent');
              citeDiv.classList.add('csl-entry');
              var biblioDiv = window.document.getElementById('ref-' + cite);
              if (biblioDiv) {
                citeDiv.innerHTML = biblioDiv.innerHTML;
              }
              popup.appendChild(citeDiv);
            });
            return popup.innerHTML;
          });
        }
      }
    });
    </script>
    

</body></html>